README.MD

```diff
@@ -49,9 +49,9 @@ let timeslots = [
 ```
 By default, one hour timeslots from 12:00AM to 11:00PM are provided.
 
-* **timeslotFormat** (`Object`): How the timeslots will be interpreted by the calendar. By default, the format goes as follows:
+* **timeslotProps** (`Object`): How the timeslots will be interpreted by the calendar. By default, the format goes as follows:
 ```js
-let timslotFormat = {
+let timeslotProps = {
```

Comment:
Same here, there was a typo in which we hve `timslotFormat`, which can be corrected with `timeslotFormat`.